test/built-ins/Temporal/Calendar/prototype/date-add/simple.js

```diff
@@ -0,0 +1,107 @@
+// Copyright (C) 2021 the V8 project authors. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-temporal.calendar.prototype.dateadd
+description: Temporal.Calendar.prototype.dateAdd
```

Comment:
Really? :P

Reply:
sorry, I do not understand this comment

Reply:
There's no hint at what is being tested here, just the full name of the API.